modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -97,6 +97,8 @@ An egress firewall has the following limitations:
 
 ifdef::ovn[]
 * A maximum of one {kind} object with a maximum of 8,000 rules can be defined per project.
+
+* If you are using the OVN-Kubernetes network plugin in Red Hat OpenShift Networking, when ingress traffic arrives at a pod affected by the egress firewall, the return ingress reply is dropped by the egress firewall. Pod to service traffic is dropped by the egress firewall when the endpoint is external, that is a pod without a classless inter-domain routing (CIDR) IP address. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2047299[*BZ#2047299*])
```

Comment:
"the return ingress reply is dropped" should only happen with shared gateway mode.

"Pod to service traffic is dropped by the egress firewall when the endpoint is external" - this needs more discussion. I think we agreed that it is not dropped, but affected by egress firewall rules (if there is no drop for that ip, it won't be dropped) in shared gateway mode, and that is expected. What may be surprising though, is in local gateway mode service traffic with external endpoint won't be affected by egress firewall rules, which may be a potential security problem. Let's talk about it after the holidays with @trozet

Reply:
we should use the user-facing name for this option (routeViaHost: false) if we need to qualify shared vs. local gateway mode.

Reply:
The first sentence makes sense to me, but the second one does not. Both cases depend on there being a default drop rule in the egress firewall spec, which is pretty common but I guess we can also explicitly state that here.

For the second sentence I thought we agreed that pod -> anything external should be dropped by egress firewall, whether that is due to accessing a service or not.

@npinaeva I just tried creating a local gw mode cluster and applying a deny egress firewall. It blocks for me:
```
trozet@fedora contrib]$ oc get svc
NAME         TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP   5m32s
[trozet@fedora contrib]$ oc get ep
NAME         ENDPOINTS         AGE
kubernetes   172.18.0.3:6443   5m34s
[trozet@fedora contrib]$ oc exec trozet3 -- curl -k https://10.96.0.1:443
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "forbidden: User \"system:anonymous\" cannot get path \"/\"",
  "reason": "Forbidden",
  "details": {},
  "code": 403
100   217  100   217    0     0  14466      0 --:--:-- --:--:-- --:--:-- 14466
[trozet@fedora contrib]$ cat ~/egressfirewall.yaml 
apiVersion: k8s.ovn.org/v1
kind: EgressFirewall
metadata:
  name: default
  namespace: default
spec:
  egress:
  - type: Allow
    to:
      cidrSelector: 1.1.1.1/32
  - type: Deny
    to:
      cidrSelector: 0.0.0.0/0
[trozet@fedora contrib]$ oc create -f ~/egressfirewall.yaml 
egressfirewall.k8s.ovn.org/default created
[trozet@fedora contrib]$ oc exec trozet3 -- curl -k https://10.96.0.1:443
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:12 --:--:--     0^C

[trozet@fedora contrib]$ oc exec trozet3 -- curl -m 2 -k https://10.96.0.1:443
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
curl: (28) Connection timed out after 2000 milliseconds
command terminated with exit code 28

```

Reply:
Right, sorry that was the wrong example (messed acl stages in my head ...again)
I think just explaining what being affected by egress firewall (instead of dropped) means should be enough together with proper gateway mode details